webapp/src/contexts/InputsContext.js

```diff
@@ -1,36 +1,47 @@
 import React, {createContext, useReducer} from 'react';
 
-const initialState = { 
+let initialState = { 
   swap: {
     from: {
       amount: "",     // Amount of tokens
       symbol: "",     // Symbol of token
       type: "",       // Native token (NEAR or ETH), ERC-20, NEP-21, ...
       logoUrl: "",    // Address of token logo image
-      tokenIndex: 1   // Index of token within token list
+      tokenIndex: 1,  // Index of token within token list
+      address: "",
+      isValid: false,
+      allowance: "",  // Required for NEP-21 -> ____ swap
```

Comment:
Can we please create classes for some common objects? Here, all the content of `from`, `to`, `pool`, .... should be instance of a class to assure we have all fields.